cmake/settings.cmake

```diff
@@ -11,6 +11,14 @@ set(AIEBU_VERSION_RELEASE 202510)
 SET(AIEBU_VERSION_MAJOR 1)
 SET(AIEBU_VERSION_MINOR 0)
 
+message("-- Host system processor is ${CMAKE_HOST_SYSTEM_PROCESSOR}")
+message("-- Target system processor is ${CMAKE_SYSTEM_PROCESSOR}")
+
+set(AIEBU_NATIVE_BUILD "yes")
+if (NOT ${CMAKE_SYSTEM_PROCESSOR} STREQUAL ${CMAKE_HOST_SYSTEM_PROCESSOR})
+  set(AIEBU_NATIVE_BUILD "no")
+endif()
```

Comment:
I would recommend using `set(AIEBU_NATIVE_BUILD TRUE)`  or `set(AIEBU_NATIVE_BUILD 1)` and then simply use `if (AIEBU_NATIVE_BUILD)` as opposed to `if (${VAR} STREQUAL ...)`